docs/adr/README.md

```diff
@@ -0,0 +1,15 @@
+# Architectural Decision Records
+
+For non-trivial architecture changes, our team uses Architectural Decision Records (ADRs) to propose, discuss, and agree
+on how/whether to update our architecture. For more information on what an ADR is, check out:
+
+1. [Michael Nygard's blog post on the topic.](https://cognitect.com/blog/2011/11/15/documenting-architecture-decisions)
+2. [The ADR GitHub organisation](https://adr.github.io)
+
+We use [this template based on Nygard's blog post](https://github.com/joelparkerhenderson/architecture_decision_record/blob/master/adr_template_by_michael_nygard.md)
+as the starting point for our ADRs.
+
+ADRs should be submitted as pull requests against this repository, which add a new ADR to this directory. ADRs are
+named based on the order in which they are approved. For example, the first ADR approved is named `ADR001.md`, the
```

Comment:
I think maybe order that they are created might be easier to manage, rather than approved. So that we don't need to rename other ADRs when one is approved. On a more general note, do we still want to try using ADRs? I know it's something that we discussed, but haven't used yet.

Reply:
Agreed on the naming, I'll change that.

I still think it's a good idea and think we should use an ADR  at least once to see how it fits.  I was thinking it might be something to try for the work I suggested on the chats, i.e. converting all actions to be a set of asynchronous events (move, play a sound, trigger an animation). I'd imagine we might end up wanting to move action definitions around, make classes for them, in short, change enough of the architecture that an ADR is warranted.

In retrospect I think the flow/typescript issue might have been a good fit, but I think we'll get a good sense of what is a good fit if we try it once.